tests/integration_async/test_workflows.py

```diff
@@ -0,0 +1,384 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2019 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import os
+
+import pkg_resources
+import time
+
+from datetime import datetime, timedelta, date
+
+from copy import deepcopy
+
+from invenio_db import db
+from invenio_search import current_search_client as es
+from invenio_workflows import ObjectStatus, start, workflow_object_class
+
+
+def generate_record():
+    """Provide record fixture."""
+    json_data = json.loads(pkg_resources.resource_string(
+        __name__,
+        os.path.join(
+            '../integration/workflows/fixtures',
```

Comment:
I think having shared fixtures is not really a good idea, even more shared between different test suits. If that fixture fits your use case, just copy it in the `integration_async/workflows/fixture` folder